src/main/kotlin/no/nav/familie/ef/sak/api/fagsak/BehandlingDto.kt

```diff
@@ -6,7 +6,7 @@ import java.time.LocalDate
 import java.util.*
 
 data class BehandlingDto (
-        val id: UUID,
```

Comment:
Trodde vi tenkte at UUID brukes internt hos oss og eksternid kun skulle benyttes for eksterne parter. Disse DTO-ene brukes av vår frontend nå - og følgelig burde vi beholde UUID på disse? Eller?

Reply:
Blir det inte en extra komplexitet att tenke på hvilke systemer som er "eksterne" vs "interne" vid konsumering av våra api:er ?  Att ha som en huskeregel att UUID er kun en db-nykkel og ikke ska eksponeres eksternt er i hvartfall før MIG en god idé. Vilka førdeler får man av bruke UUID på frontend?

Reply:
P.S Både jag och Martine tenkte likt om gå øver til å bruke long i frontend XD

Reply:
hmm, spørs om vi skal ta en rask avklaring av dette på tirsdag? Min tanke var iallefall at vi skiller på:
1) Våre systemer som bruker UUID 
2) Eksterne systemer (JoArk, Oppgave, Økonomi, Infotrygd, Statistikk et.al) som bruker EksternId